### PR TITLE
PS-56 add parcel display page and search parcel component

### PR DESCRIPTION
**Pull request by ciliakas**
For now the parcel table display basically half mock data, but it will be easy to add new things to display and substitute with real data.
Also the endpoint calls will be fixed once required endpoints will be available

### CODE REVIEW

File: dbilo-siuntos-frontend/src/types/parcels.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+export interface Parcel {
+  id: number;
+  trackingNumber: string;
+  size: ParcelSize;
+  shipperID: number;
+  shipmentDate: string;
+  shippingAddress: Address | null;
+  deliveryDate: string;
+  deliveryAddress: Address | null;
+}
```
Comment (Stokalas) [inline dbilo-siuntos-frontend/src/types/parcels.ts:13]: This is very nice, will use it in my task with POST, cool

File: dbilo-siuntos-frontend/src/types/address.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  addressLine2: string;
+  postalCode: string;
+  country: string;
+}
+// public int Id { get; set; }
+// public string City { get; set; }
+// public string AddressLine1 { get; set; }
+// public string? AddressLine2 { get; set; }
+// public string PostalCode { get; set; }
+// public string Country { get; set; }
```
Comment (Stokalas) [inline dbilo-siuntos-frontend/src/types/address.ts]: Is this needed?

File: dbilo-siuntos-frontend/src/components/router/Header.tsx [hunk 3]
```diff
@@ -28,8 +28,7 @@ export const Header: React.FC = () => {
 
   const leftRoutes = [
     { ...ROUTES.HOMEPAGE, renderIcon: () => <HomeIcon /> },
-    { ...ROUTES.ABOUT, renderIcon: () => <InfoIcon /> },
-    { ...ROUTES.CREATE_PARCEL, renderIcon: () => <InfoIcon /> },
+    { ...ROUTES.CREATE_PARCEL, renderIcon: () => <SendIcon /> },
```
Comment (Stokalas) [inline dbilo-siuntos-frontend/src/components/router/Header.tsx:31]: Can you shortly explain this `renderIcon` prop?

**Review by Stokalas — APPROVED** [review]
I like it visually and seems to work as ex[ected, nice job

File: dbilo-siuntos-frontend/src/components/router/Header.tsx (hunk 3 above)
Comment (ciliakas) [reply dbilo-siuntos-frontend/src/components/router/Header.tsx:31]: Basically it's just the icon that will be used in the hamburger menu. Now I can't exactly remember why I'm passing it as a function :D

**Pull request merged**